src/components/figCaption.tsx

```diff
@@ -76,11 +77,11 @@ const captionElement = (format: Format) => (node: Node, key: number): ReactNode
     const children = Array.from(node.childNodes).map(captionElement(format));
     switch (node.nodeName) {
         case 'STRONG':
-            return <h2 css={captionHeadingStyles} key={key}>{children}</h2>;
+            return <strong css={captionHeadingStyles} key={key}>{children}</strong>;
```

Comment:
The choice of `h2` here was for semantic reasons (plus they're `h2` elements on dotcom). If the aim is to get inline styles can we use `display: inline` instead?